qiskit/ignis/verification/randomized_benchmarking/fitters.py

```diff
@@ -416,8 +415,10 @@ def fit_data(self):
             if alpha_guess < 1.0:
                 fit_guess[1] = alpha_guess
 
-            fit_guess[0] = ((y0 - fit_guess[2]) /
-                            fit_guess[1]**self._cliff_lengths[patt_ind][0])
+            if y0 > fit_guess[2]:
+                fit_guess[0] = ((y0 - fit_guess[2]) /
+                                fit_guess[1]**self._cliff_lengths[patt_ind][0])
+                
```

Comment:
There's a lint error here, can't be any spaces on this line

Reply:
Fixed